src/SFA.DAS.ApprenticeFeedback.Data/ApprenticeFeedbackDataContext.cs

```diff
@@ -15,6 +15,7 @@
 using SFA.DAS.ApprenticeFeedback.Domain.Interfaces;
 using SFA.DAS.ApprenticeFeedback.Data.Configuration;
 using SFA.DAS.ApprenticeFeedback.Domain.Configuration;
+using SFA.DAS.ApprenticeFeedback.Domain.Models;
```

Comment:
I think if you lose this using statement, you can undo most of the changes to this file, keeping the IExitSurveyContext and Exit Survey additions ( only entities should be in here anyway, not models )